tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -891,7 +891,7 @@ def percentage_of_reoccurring_datapoints_to_all_datapoints(x):
 
 @set_property("fctype", "simple")
 @set_property("input", "pd.Series")
-def percentage_of_reoccurring_values_to_all_values(x):
+def percentage_of_reoccurring_datapoints_to_all_datapoints(x):
     """
     Returns the ratio of unique values, that are present in the time series
     more than once.
```

Comment:
Could you also fix the comment further down (And also in the other function?). It still references the old function name

Reply:
Missed that! Fixed now